frontend/src/pages/projects/notebook/useRelatedNotebooks.ts

```diff
@@ -1,19 +1,14 @@
 import * as React from 'react';
 import { NotebookKind } from '~/k8sTypes';
-import { getNotebookPVCNames, getNotebookSecretNames } from '~/pages/projects/pvc/utils';
+import { getNotebookPVCNames } from '~/pages/projects/pvc/utils';
 import { ProjectDetailsContext } from '~/pages/projects/ProjectDetailsContext';
-import { DATA_CONNECTION_PREFIX } from '~/api';
 import { getNotebookPVCMountPathMap } from './utils';
 
 export enum ConnectedNotebookContext {
   /** What PVCs are already connected to notebooks */
   EXISTING_PVC = 'pvc',
   /** What PVCs are allowed to be disconnected from notebooks */
   REMOVABLE_PVC = 'removable-pvc',
-  /** What Data Connections are already connected to notebooks */
-  EXISTING_DATA_CONNECTION = 'data-connection',
-  /** What Data Connections are possible to connect to notebooks */
-  POSSIBLE_DATA_CONNECTION = 'data-connection-possible',
 }
```

Comment:
Interesting -- I feel this probably is needing to be removed entirely... we have a new PVC infrastructure 🤔 

@Gkrumbach07 do you know off the top of your head?

I'll have to check this PR out and do code surfing to see what's going on (cc @emilys314 if you get to it before me)

Reply:
You have broken the other instances of this -- I think this needs to stay too -- because this functionality has value today and I think we are losing that.